mods/ts/chrome/editor.yaml

```diff
@@ -0,0 +1,971 @@
+			Align: Center
+			Font: Bold
+		Label@TITLE_LABEL:
+			X: 10
+			Y: 60
+			Width: 95
+			Height: 25
+			Align: Right
+			Text: label-save-map-panel-title
+		TextField@TITLE:
+			X: 110
+			Y: 60
+			Width: 220
+			MaxLength: 50
+			Height: 25
+		Label@AUTHOR_LABEL:
+			X: 10
+			Y: 95
+			Width: 95
+			Height: 25
+			Align: Right
+			Text: label-save-map-panel-author
+		TextField@AUTHOR:
+			X: 110
+			Y: 95
+			Width: 220
+			MaxLength: 50
+			Height: 25
+		Label@VISIBILITY_LABEL:
+			X: 10
+			Y: 130
+			Width: 95
+			Height: 25
+			Align: Right
+			Text: label-save-map-panel-visibility
+		DropDownButton@VISIBILITY_DROPDOWN:
+			X: 110
+			Y: 130
+			Width: 220
+			Height: 25
+			Text: dropdownbutton-save-map-panel-visibility-dropdown
+		Label@DIRECTORY_LABEL:
+			X: 10
+			Y: 165
+			Width: 95
+			Height: 25
+			Align: Right
+			Text: label-save-map-panel-directory
+		DropDownButton@DIRECTORY_DROPDOWN:
+			X: 110
+			Y: 165
+			Width: 220
+			Height: 25
+		Label@FILENAME_LABEL:
+			X: 10
+			Y: 200
+			Width: 95
+			Height: 25
+			Align: Right
+			Text: label-save-map-panel-filename
+		TextField@FILENAME:
+			X: 110
+			Y: 200
+			Width: 105
+			Height: 25
+			Type: Filename
+		DropDownButton@TYPE_DROPDOWN:
+			X: 220
+			Y: 200
+			Width: 110
+			Height: 25
+		Button@SAVE_BUTTON:
+			X: 80
+			Y: PARENT_BOTTOM - 45
+			Width: 120
+			Height: 25
+			Text: button-save-map-panel
+			Font: Bold
+		Button@BACK_BUTTON:
+			X: 210
+			Y: PARENT_BOTTOM - 45
+			Width: 120
+			Height: 25
+			Text: button-cancel
+			Font: Bold
+			Key: escape
+
+ScrollPanel@MAP_SAVE_VISIBILITY_PANEL:
+	TopBottomSpacing: 5
+	ItemSpacing: 5
+	Width: 220
+	Height: 55
+	Children:
+		Checkbox@VISIBILITY_TEMPLATE:
+			X: 5
+			Width: PARENT_RIGHT - 29
+			Height: 20
+			Font: Regular
+
+Container@EDITOR_ROOT:
+	Logic: LoadMapEditorLogic
+	Children:
+		LogicKeyListener@GLOBAL_KEYHANDLER:
+			Logic: MusicHotkeyLogic, ScreenshotHotkeyLogic, MuteHotkeyLogic, EditorQuickSaveHotkeyLogic
+				StopMusicKey: StopMusic
+				PauseMusicKey: PauseMusic
+				PrevMusicKey: PrevMusic
+				NextMusicKey: NextMusic
+				TakeScreenshotKey: TakeScreenshot
+				MuteAudioKey: ToggleMute
+				QuickSaveKey: EditorQuickSave
+		LogicKeyListener@WORLD_KEYHANDLER:
+			Logic: ResetZoomHotkeyLogic
+				ResetZoomKey: ResetZoom
+		Container@WORLD_ROOT:
+		Container@MENU_ROOT:
+		TooltipContainer@TOOLTIP_CONTAINER:
+
+Container@EDITOR_WORLD_ROOT:
+	Logic: LoadIngamePerfLogic, MapEditorLogic, ActorEditLogic, MapOverlaysLogic, MapEditorSelectionLogic
+		ToggleGridOverlayKey: EditorToggleGridOverlay
+		ToggleBuildableOverlayKey: EditorToggleBuildableOverlay
+		ToggleMarkerOverlayKey: EditorToggleMarkerOverlay
+	Children:
+		LogicKeyListener@OVERLAY_KEYHANDLER:
+		Container@PERF_ROOT:
+		EditorViewportController@MAP_EDITOR:
+			Width: WINDOW_RIGHT
+			Height: WINDOW_BOTTOM
+			TooltipContainer: TOOLTIP_CONTAINER
+			TooltipTemplate: SIMPLE_TOOLTIP
+			SelectionAltOffset: 0,1
```

Comment:
This line for `SelectionAltOffset: 0,1` is the only diff from the common UI? Seems unfortunate to need to copy paste the whole thing for this. Flagging in case there's any nicer way to do it, but appreciate there may not be.

Is it an isometric map thing? If so making a tweak in code to adjust automatically would seem sensible.

Reply:
It is the only thing currently. This refactor should allow adding new brushes that are exclusive to isometric mods. Though for quite a few things first I feel like a refactor like #20582 should happen as well, separating rectangular and isometric maps into separate classes

Reply:
If it's just that we need to add a correction offset in isometric, then I don't see why the existing brush couldn't just apply that fixup?

Reply:
I removed the split, although I'd really prefer to have the TS map editor separate